src/scene/export.ts

```diff
@@ -57,6 +58,7 @@ export const exportToCanvas = async (
     scale,
     rc: rough.canvas(canvas),
     canvas,
+    canvasUi,
```

Comment:
seeing this, we'll actually need to export the `renderCanvasContent` so that export functions don't need to be creating and passing a canvas they don't care about (`canvasUi`).

If we later have `_renderScene()` do some data preprocessing before calling `renderCanvasContent()` (hence making it harder to use `renderCanvasContent()` directly in export functions), we'll figure out what to do, then.

Reply:
then we have to make it optional `canvasUi`, in that case...!!?

Reply:
no, in the export functions we'll use `renderCanvasContent()` directly

Reply:
ok, that will work